lib/test-worker.js

```diff
@@ -1,38 +1,56 @@
 'use strict';
-/* eslint-disable import/order */
-const isObj = require('is-obj');
-const process = require('./process-adapter');
 
-const opts = process.opts;
-const testPath = opts.file;
+// Check if the test is being run without AVA cli
+{
+	/* eslint-disable import/order */
+	const path = require('path');
+	const chalk = require('chalk');
+
+	const isForked = typeof process.send === 'function';
```

Comment:
`karma-ava` is pretty experimental though. We don't have a stable API so IMHO it's fine if this breaks `karma-ava`.

If necessary this could live in its own file. It's odd in `process-adapter`. Heck it's a bit odd to have it in `test-worker` versus just `main`.